index/scorch/snapshot_rollback_test.go

```diff
@@ -84,14 +93,17 @@ func TestIndexRollback(t *testing.T) {
 		_ = readerSlow.Close()
 	}()
 
-	// fetch rollback points available as of here
-	rollbackPoints, err := sh.RollbackPoints()
-	if err != nil || len(rollbackPoints) != 1 {
+	// fetch rollback points after first batch
+	rollbackPoints, err = sh.RollbackPoints()
+	if err != nil {
 		t.Fatal(err, len(rollbackPoints))
 	}
+	if len(rollbackPoints) == 0 {
+		t.Fatalf("expected some rollbackPoints, got none")
+	}
 
-	// set this as a rollback point for the future
-	rollbackPoint := rollbackPoints[0]
+	// remember earliest rollback point
+	rollbackPoint := rollbackPoints[len(rollbackPoints)-1]
```

Comment:
this seems to be a change in the semantics, previously we grabbed the most recent rollback point, now we grab the oldest

Reply:
yah - will go back to the original approach of the "most recent rollback point"